pom.xml

```diff
@@ -60,31 +52,32 @@
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
 
-        <spring-cloud.version>Greenwich.RC2</spring-cloud.version>
-
-        <spring.boot.version>2.1.1.RELEASE</spring.boot.version>
+        <spring-boot.version>2.1.1.RELEASE</spring-boot.version>
+        <spring-cloud.version>Finchley.RELEASE</spring-cloud.version>
```

Comment:
see the compatibility matrix : http://start.spring.io/actuator/info

"Greenwich.RC2":"Spring Boot >=2.1.0.RELEASE and <2.1.2.BUILD-SNAPSHOT"

--> we should stay with Greenwich